Dockerfile

```diff
@@ -6,13 +6,15 @@ RUN apt-get update && apt-get install -y gcc
 WORKDIR /scripts
 
 # copy over the requirements.txt and install dependencies
-COPY requirements.txt .
-
-RUN pip install --no-cache-dir -r requirements.txt
+COPY setup.py requirements.txt /opt/segmentation/
+RUN pip install -r /opt/segmentation/requirements.txt
 
 # copy the scripts over
-COPY segmentation /segmentation
+COPY segmentation /opt/segmentation/segmentation
 COPY scripts /scripts
 
+# Install the package via setup.py
+RUN pip install /opt/segmentation
+
```

Comment:
I think that this might affect the "in docker's scope" checking being added in PR #110 (see issue #113), but I'm not 100% sure of that

Reply:
I don't think this will affect that. Changing the folder that segmentation is located in within the docker shouldn't affect the logic about relative paths, right? Your code just looks at the cwd, which looks like it's relative to the docker container 
![image](https://user-images.githubusercontent.com/13770365/86671168-3dc6b680-bfaa-11ea-9c42-7068175eb82e.png)